jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -1843,13 +1901,73 @@ Function Start-Migration {
         [Parameter(ParameterSetName = "form")][Object]$inputObject)
 
     Begin {
+        $progressCount = 16
```

Comment:
Where are the 16 items? I feel like this should be something dynamic, maybe we define an ordered list of steps in `Start-Migration` and pull from that list here. 

Editing this list seems really painful in the current implementation. I'd need to count the number of times we have a help message to display, then keep track of that number and go back to this function and manually update this number. Seems like we can just define an ordered hash table and pull the message contents from there?